### ISSUE TITLE
`precompiled_apple_resource_bundle` still requires `common_tool_attributes` that's not exist in `rules_apple` newest version

### ISSUE THREAD

**Issue by kennethvl** [bazel-ios/rules_ios#779]
Hi there,

We're currently updating our `rules_apple` version to the newest v3.0.0. But our build is failing due to we're using the `precompiled_apple_resource_bundle` rule from `rules_ios` in our app.

The error log is:
```
(16:40:14) ERROR: Traceback (most recent call last):

	File "/private/var/tmp/_bazel_ssm-user/.../external/build_bazel_rules_ios/rules/precompiled_apple_resource_bundle.bzl", line 229, column 21, in <toplevel>

		rule_factory.common_tool_attributes,

Error: 'struct' value has no field or method 'common_tool_attributes'

Available attributes: create_apple_rule, create_apple_test_rule
```

When I look at the `rule_factory.bzl` file on `rules_apple`, I see that the `common_tool_attributes` is not there anymore, as seen here: https://github.com/bazelbuild/rules_apple/blob/master/apple/internal/rule_factory.bzl#L172-L175. 

Although in the `precompiled_apple_resource_bundle.bzl` file, it's expected that the `common_tool_attributes` still exists, as seen here: https://github.com/bazel-ios/rules_ios/blob/master/rules/precompiled_apple_resource_bundle.bzl#L229.

Can someone help about this issue? Thanks!

**Comment by jszumski** [comment]
Support is being added here: https://github.com/bazel-ios/rules_ios/pull/773

**Comment by kennethvl** [comment]
I see that the support has just merged. Can I assume someone will create a new release soon after?

**Comment by lucasromanomr** [comment]
Adiciona no WORKSPACE e funcionou. Acredito que vai ser ajustado no futuro

```
load(
    "@build_bazel_rules_ios//rules:repositories.bzl",
    "rules_ios_dependencies",
    "rules_ios_dev_dependencies"
)

rules_ios_dependencies()
rules_ios_dev_dependencies()
```

I was having problems, I think that after blzmod came out it ended up missing some things in the documentation.

**Comment by lucasromanomr** [comment]
I believe that now just update to the latest version of the rules :)

**Comment by luispadron** [comment]
Fixed in #773
